config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   root "home#index"
   post "check_translation" => "home#check_translation"
-  resources :cards
+  #get "login" => "users#login"
+  get 'login' => 'user_sessions#new', :as => :login
+  post 'login', to: 'user_sessions#create'
+  post 'logout' => 'user_sessions#destroy', :as => :logout
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.